skycultures/arabic_lunar_stations/constellation_names.eng.fab

```diff
@@ -0,0 +1,55 @@
+0199 "الفرقدان" _("The Two Oryx Calves")
+0299 "فأس الرحى" _("The Axe Of The Grindstone")
+0399 "بنات نعش" _("Daughters Of Na'sh")
+0410 "القفزة الأولى" _("The First Leap")
+0420 "القفزة الثانية" _("The Second Leap")
+0430 "القفزة الثالثة" _("The Third Leap")
+0699 "الذئبان" _("The Two Wolves")
+0799 "العوائذ" _("The Mother Camels")
+1499 "قصعة المساكين" _("The Bowl Of The Poor")
+1901 "رأس الناقة" _("The Camel Head")
+1902 "عنق الناقة" _("The Camel Neck")
+1903 "يدا الناقة" _("The Camel Front Legs")
+1999 "الناقة" _("The She-Camel")
+2501 "الكرب - النعام" _("The Bucket Rope Knot - The Well Poles")
+2502 "الفرغ المقدم" _("The Front Bucket Mouth")
+2504 "الفرغ المؤخر" _("The Rear Bucket Mouth")
+2550 "الرشاء" _("The Fish")
+2602 "الشرطان" _("The Two Signs")
+2603 "البطين" _("The Little Abdomen")
+2604 "الثريا" _("Al-Thurayya")
+2801 "الذراع المقبوضة" _("Retracted Arm")

```

Comment:
Most other names are with article. Is there a reason to not write "The Retracted Arm" here?

Reply:
No reason. Originally I had "Retracted Arm [Of The Lion]" to clarify the name. The native name is only "The Retracted Arm".

Fixed.